### PR TITLE
Array of strings on ShellCommand to allow paths with spaces

### PR DESCRIPTION
**Pull request by javiermanzano**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -20,9 +20,9 @@ allprojects {
 
 ext {
     compileSdkVersion = 22
-    buildToolsVersion = '22'
+    buildToolsVersion = '21.1.2'
```
Comment (hiteshsondhi88) [inline build.gradle]: can you please change this to 22

**Comment by hiteshsondhi88** [comment]
This PR will affect the current users, what do you think about creating polymorphic methods and leave the current users unaffected ?

**Comment by JovinPJ** [comment]
How can i get these changes? i have the same issue with spaces in filepath

**Comment by rafaelekol** [comment]
I also need this commit, can you advice how to import this changes to our project.

**Comment by JovinPJ** [comment]
Thanks hiteshsondhi88, worked for me :)

**Comment by ab-helly** [comment]
@JovinPJ how did you get these changes with gradle?

**Comment by ab-helly** [comment]
@JovinPJ how did you get these changes with gradle?

**Comment by JovinPJ** [comment]
@ab-helly 
please check the link http://hiteshsondhi88.github.io/ffmpeg-android-java/
Actually, what i've done is downloaded the '.aar' and included in my lib folder
and added the dependency like
'compile(name:'FFmpegAndroid', ext:'aar')'.
'FFmpegAndroid' was name of my aar file
There is also option like , 
compile 'com.github.hiteshsondhi88.libffmpeg:FFmpegAndroid:0.2.5' for adding dependency

One drawback ,i noticed is that the apk size become very large(>22mb).

Thanks

**Comment by ab-helly** [comment]
@JovinPJ thanks, aar should work for me. 0.2.5 still doesn't contain these changes

**Comment by brambo48** [comment]
How can I use this commit?

**Comment by mattsff** [comment]
I downloaded aar file and it's not working yet, do you know how can I get it ? Only compiling the fork? Thanks!

**Comment by JovinPJ** [comment]
please let me know , what is your issue.

ffmpeg.execute(cmd, new ExecuteBinaryResponseHandler() {

}

here 'cmd' is a string array.



did you able to compile this?


Thanks

**Comment by mattsff** [comment]
I could do it donwloading all project and put it as module. I couldn't through gradle or AAR file. 
It works now, thanks !

**Comment by kschick** [comment]
@JovinPJ How did you get this change from the aar? The aar linked to from that site is still version 0.2.5.

**Comment by Poeticode** [comment]
Yeah, the current .aar in this repository is using an old version. You'll have to pull down the repository and open it in Android Studio, then run the assembleRelease gradle script via the project root's gradle file.

screen shot 2016-03-29 at 3 18 42 pm

If it runs successfully, you'll find the updated .aar at build/outputs/aar/FFmpegAndroid-release.aar

I personally had to undo the String[] changes in this commit and swapped out the FFmpeg binaries with the more recent ones from PR #70

**Comment by JovinPJ** [comment]
Hi ,
 sorry for the delay, if you give me your mail id i'll send the aar file

**Comment by lekhanhduybkhn** [comment]
Hi JovinPJ  there is my email khanhduy.8891@gmail.com. Could you send me arr file. so I try download aar and gradle but file name with space is don't work for me

**Comment by symphonyrecords** [comment]
[WhiteSpace in file path](https://stackoverflow.com/a/51594776/6017001)

**Pull request merged**